RFUSB_to_DB15/drivers.cpp

```diff
@@ -68,6 +68,7 @@ void setupController(uint16_t vid, uint16_t pid, HIDController *controller) {
 
     case VID_HONCAM:
       if (pid == PID_HONCAM) setupHoncam(controller);
+      if (pid == PID_FUSION_PS4) setupPS4(controller);
```

Comment:
generally `else if` or a `case` statement should be used here. It shouldn't be possible for pid to match both though so it shouldn't be too much of an issue